Unity/Assets/SidebarButtonHandler.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class SidebarButtonHandler : MonoBehaviour
+{
+    public Button sidebarButton;
+
+    void Start()
+    {
+        Button btn = sidebarButton.GetComponent<Button>();
```

Comment:
Is this code connected to a button in the Scene?

Reply:
Yes, it is connected to every button in the sidebar and will then dynamically show, which button has been pressed in the console.

Reply:
Can you do a clean clone of the branch you've pushed to see how it works on our end? I think its missing a file and that's why its not working

Reply:
Sure, let me try

Reply:
Apparently the script hadn't saved, when I assigned it to the buttons. I reassigned it, closed and opened Unity again. It worked this time. Can you check, please?

Reply:
![image](https://user-images.githubusercontent.com/95181353/224540399-6f130c53-8e9b-4866-a75d-89df7b8c7d09.png)

im not sure what's wrong with it, if I add the script again then it works

Reply:
I guess we'll check on monday